coverage.cfg

```diff
@@ -17,3 +18,12 @@ exclude_lines =
     pragma: no${PLATFORM}
     pragma: skipif_${PY_39}_py39
     pragma: skipif_${PY_36}_py36
+fail_under = 100
+show_missing = true
+
+[html]
+directory = htmlcov/${TOX_ENV_NAME}
+title = aiosmtpd coverage for ${TOX_ENV_NAME}
+
+[xml]
+output = coverage-${INTERP}.xml
```

Comment:
Because of some version-specific nocover's, better ensure that the coverage XML for testenvs are separate.